### PR TITLE
Featuer/ab2d 606 filter eob part d out

### PR DESCRIPTION
**Pull request by lailasharshar**
### Summary
Filters out part D claims which makes sense since the point of this project is to take part A & B
claims and hand them over to part D providers.

### CODE REVIEW

File: filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java [hunk 1]
```diff
@@ -110,4 +109,64 @@ public void testItemValues() {
...
+    void demo5() {
+        printItOut("eobdata/EOB-for-Inpatient-Claims.json");
+    }
+
+    @Test
+    void demo6() {
+        printItOut("eobdata/EOB-for-Outpatient-Claims.json");
+    }
+
+    @Test
```
Comment (dkrylovsb) [inline filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java]: Do we need these?

File: filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java [hunk 2]
```diff
@@ -110,4 +109,64 @@ public void testItemValues() {
...
+        printItOut("eobdata/EOB-for-Part-D-Claims.json");
+    }
+
+    @Test
+    void demo8() {
+        printItOut("eobdata/EOB-for-SNF-Claims.json");
+    }
+
+    @Test
+    void isPlanB() {
```
Comment (dkrylovsb) [inline filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java]: You probably meant `isPlanD`

Comment (lailasharshar) [reply filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java]: Maybe one for demonstrative purposes.

File: filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java (hunk 1 above)
Comment (lailasharshar) [reply filter/src/test/java/gov/cms/ab2d/filter/ExplanationOfBenefitsTrimmerTest.java]: Maybe one for demonstrative purposes.

**Pull request merged**